src/main/webapp/Filters.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
```

Comment:
Let's not add this file until it is being used, would be better to keep it in the local change if you plan to use it in a followup PR.

Reply:
Ah okay just un-added it.